delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/acls/AclsImpl.scala

```diff
@@ -174,4 +174,27 @@ object AclsImpl {
       acls  = AclsImpl.apply(agg, eventLog, index)
       _    <- UIO.delay(startIndexing(config, eventLog, index, acls))
     } yield acls
+
+  /**
+    * Constructs an [[AclsImpl]] instance.
+    *
+    * @param config       ACLs configurate
+    * @param permissions  [[Permissions]] instance
+    * @param eventLog     the event log
+    */
+  final def apply(
```

Comment:
We should only need one apply method no ?

Reply:
Removed the other apply method.